api/v1alpha1/openapi.yaml

```diff
@@ -3936,6 +3936,9 @@ components:
         operatingSystem:
           type: string
           description: The Operating System reported by the device.
+        agentVersion:
```

Comment:
One question, do we think the agent will be independent from the OS image? If no should this status be part of `os:`?

Reply:
AFAIK we can build agent-0.3.0 and -0.4.0 for RHEL9, so yes, it will be different

Reply:
Right, I guess the question rephrased is can you upgrade the agent without a new OS image?

Reply:
If I understand you correctly: I can build different agents into the same OS image.
So Architecture (runtime.GOARCH) and OperatingSystem (runtime.GOOS) describes OS, but agentVersion describes agent. It can be different version of agent running on the same ARCH-OS.
So I think agent version is not connected to OS version, and I'd not mix them

Reply:
@hexfusion: You’re correct that with bootc/rpm-ostree we assume users bake the agent into the OS image, but we may relax that constraint. Say, when the agent is deployed on an rpm/deb package-based system or containerized.

Reply:
fair